packages/account/src/Components/file-uploader-container/file-uploader-container.tsx

```diff
@@ -1,23 +1,78 @@
-import React from 'react';
-import { Text } from '@deriv/components';
-import { Localize } from '@deriv/translations';
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+//@ts-nocheck [TODO] - Need to fix typescript errors in this file
+import React, { useEffect, useState } from 'react';
```

Comment:
if we need React here to make this page working outside of account, so maybe we don't need separate import for hooks?

Reply:
Yes. Will make the change. Thank you